### DISCUSSION TITLE
Unable to load weather and telemetry data for seasons before 2018

### DISCUSSION THREAD

**Discussion by laowangwangz** [theOehrly/Fast-F1#647] [Q&A] [answered]
I was trying to obtain historical laps and weather data, but it seems fastf1 is unable to load the data for seasons before 2018? 

Here is an example of trying to load data for 2017 Australian Grand Prix:

![1729523739193](https://github.com/user-attachments/assets/09a82bb3-c26c-4a9a-bbd8-fc646f049c43)

The same code works for 2018 Australian Grand Prix. Has anyone encountered this issue and solved it? Really appreciate the help!

**Comment by theOehrly** [comment]
Hi, FastF1 currently does not support loading this data for seasons before 2018. The only data that is available before 2018 are session results (with limited data as well).

**Reply by laowangwangz** [reply]
Thanks a lot! I was unable to load laps and weather and telemetry data for 2022 as well. I was able to load them for 2018, 2019, 2020, 2021, and 2023, though. Is there any particular reason?

**Reply by Casper-Guo** [reply, marked as answer]
There's been a bit of an issue with the 2022 season in general. A mirror server has been set up to serve those requests. If you are not on version 3.4.0+ already I recommend updating and see if you get any improvement. In my case, that mostly resolved the issue

**Reply by laowangwangz** [reply]
Updating solved my problem. Thanks a ton!!
